Makefile

```diff
@@ -45,3 +45,14 @@ clean:
 
 nopyc:
 	find . -name \*.pyc -delete
+
+API_DEFINITIONS_SHA=$(shell git log --oneline | grep Regenerated | head -n1 | cut -d ' ' -f 5)
```

Comment:
Should we add a prefix like `apidefs` to this to distinguish it from other SHA's?

Reply:
I think so.. what do you think about `twilio/twilio-python:apidesf-<sha>`?